src/main/java/ledger/user_interface/ui_controllers/window/LoginPageController.java

```diff
@@ -111,9 +121,16 @@ private void login() {
         }
         try {
             DbController.INSTANCE.initialize(this.filePath, pwd);
+            Preferences preferences = Preferences.userRoot().node("LedgerSoftware");
+            preferences.put(LAST_DATABASE_FILE_KEY,this.filePath);
+            preferences.flush();
             Startup.INSTANCE.newStage(new Scene(new MainPageController()), "Ledger");
+
+
         } catch (StorageException e) {
             this.setupErrorPopup("Unable to connect to database", e);
+        } catch (BackingStoreException e){
+            this.setupErrorPopup("Unable store last Db file", e);
```

Comment:
I don't know if the user cares about this error. Might be more appropriate in System.err